sentry-ruby/lib/sentry/hub.rb

```diff
@@ -150,7 +150,7 @@ def capture_event(event, **options, &block)
         configuration.log_debug(event.to_json_compatible)
       end
 
-      @last_event_id = event&.event_id
+      @last_event_id = event&.event_id unless event.is_a?(Sentry::TransactionEvent)
```

Comment:
@sl0thentr0py this is the reason I want to do the inheritance-removal refactor. they are basically 2 types of events but yet they're both `Sentry::Event`.